wikidatarefisland/data_access/storage.py

```diff
@@ -16,8 +16,8 @@ def getLines(self, file_):
         with open(path, 'r') as f:
             for line in f:
                 try:
-                    yield json.loads(line)
-                except json.JSONDecodeError:
+                    yield json.loads(line[:-2])
```

Comment:
I'm confused regarding this, why would we want to trim the last two chars?

Reply:
This is why it's still WIP mostly; so this is the hack to make to storage work with the normal production dumps

Reply:
I honestly don't see a way around it. The way dumps are built is a mess.
I recommand removing "\n" from the line and then using str.trim(',')

Reply:
Well, this can't be merged as is right? It will break other pipes parsing jsonlines?

Reply:
for now, let's put a try/except clause and then we separate the two by "readDumpLines()" and readLines()

Reply:
Also removing "\n" from the line and then using str.trim(',') would work just fine for both jsonlines and dumps btw.